Specs/DateTime/Dutch/DateParser.json

```diff
@@ -1243,19 +1243,19 @@
           }
         },
         "Start": 11,
-        "Length": 24
+        "Length": 22
       }
     ]
   },
   {
-    "Input": "wie heb ik een paar dagen geleden een email gestuurd",
+    "Input": "wie heb ik enkele dagen geleden een email gestuurd",
```

Comment:
`een paar dagen geleden` is also a valid Dutch sentence and frequently used. Is this one covered by another spec already?

Reply:
the Input was updated because the original English case uses "few" that has a different resolution than "a couple of". There are other test cases where "een paar" is used.

Reply:
But do we have a separate case with "een paar"? We should at least keep it and mark temporarily as NotSupported if not yet passing. We should never remove spec cases that are plausible in a target language.

Reply:
in DurationParser there are cases with "en paar" (e.g. "Ik ben een paar dagen weg.").
There are no such cases in DateParser (neither in Dutch nor in English)